migrator/pom.xml

```diff
@@ -0,0 +1,56 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.jenkov</groupId>
```

Comment:
For the group, you'll want the same ID as the metadata/ stuff: com.google.i18n.libphonenumber
Also, Snapshot? 0.1?
None of this matters yet, but adding a TODO to come back and sort this out later might be a good idea.